build.ps1

```diff
@@ -98,4 +98,3 @@ if (Test-Path $catalogFilePath) {
    # Delete previous catalog file
    Remove-Item $catalogFilePath -Confirm:$false
 }
-New-FileCatalog -Path $OutputDir -CatalogFilePath $catalogFilePath | Out-Null
```

Comment:
Optional suggestion: Maybe we can remove the whole step **Prepare Module for Publishing** although the condition for removing the previous catalog file will never be invoked with the removal of the **NewFileCatalog** call.